src/commands/money.js

```diff
@@ -101,10 +106,10 @@ export default class Money {
 
         this.BASE = data.base
         this.RATES = data.rates
-        this.LAST_UPDATE = data.timestamp * 1000
+        this.LAST_UPDATE = data.timestamp           *                                                1000
```

Comment:
Line is longer than allowed. (105 > 100)

*Origin: LineLengthBear, Section: `all.linelength`.*